analytics_data_api/management/commands/tests/test_generate_fake_course_data.py

```diff
@@ -13,7 +15,8 @@ def testNormalRun(self):
             'generate_fake_course_data',
             f"--num-weeks={num_weeks}",
             "--no-videos",
-            "--course-id", course_id
+            "--course-id", course_id,
+            "--database", 'analytics',
```

Comment:
I would like to run this command with the `analytics_v1` argument as well, but because the `AnalyticsModelsRouter` prevents migrations from running on the v1 database, the tables do not exist in the database. This causes failures.

Reply:
this test demonstrates that you can give it a DB name and generate fake data so a run on a different DB name doesn't prove much more